doc/src/docs/InputOutputReference/01d-InputOutputReference.md

```diff
@@ -8392,9 +8392,221 @@ This output field is the average electricity consumption rate of the DX coil com
+
+#### Field: Rated Evaporator Inlet Air Wet-Bulb Temperature
+
+This numeric field defines the evaporator inlet air wet-bulb temperature, in degrees Celsius, that corresponds to rated coil performance (heating capacity, COP and SHR). Values must be greater than 5 &deg;C. If this field is left blank, the default value is 13.5 &deg;C.
+
+#### Field: Rated Condenser Water Temperature
+
+This numeric field defines the condenser inlet water temperature, in degrees Celsius, that corresponds to rated coil performance (heating capacity, COP and SHR). Values must be greater than 25 &deg;C. If this field is left blank, the default value is 57.5 &deg;C.
+
+#### Field: Rated Evaporator Air Flow Rate
+
+This numeric field defines the evaporator air volume flow rate in cubic meters per second at rated conditions. Values must be greater than 0. If this field is left blank or autocalculated   (field value = **autocalculate**), the default value is 5.035E-5 m<sup>3</sup>/s/W (31.25 cfm/MBH) multiplied by the Rated Heating Capacity specified above. When autocalculating the rated evaporator air volumetric flow rate, a zone sizing object is not required.
+
+#### Field: Evaporator Fan Power Included in Rated COP
+
+This choice field specifies if evaporator fan power is included in the rated COP defined above. This input impacts the calculation of compressor electric power and total air cooling provided by the evaporator for each simulation timestep. If Yes is selected, the evaporator fan power is subtracted from the total electric heating power when calculating total evaporator cooling capacity. If No is selected, it is assumed that the total heating power does not include evaporator fan power. If this field is left blank, the default is Yes. See the Engineering Reference section for `Coil:WaterHeating:AirToWaterHeatPump:\*` for further details.
+
+#### Field: Evaporator Air Inlet Node Name
+
+This alpha field defines the name of the air node from which the evaporator coil draws its inlet air.
+
+#### Field: Evaporator Air Outlet Node Name
+
+This alpha field defines the name of the air node to which the evaporator coil sends its outlet air.
+
+#### Field: Crankcase Heater Capacity
+
+This numeric field defines the compressor's crankcase heater capacity in Watts. The crankcase heater only operates when the compressor is off and the air surrounding the compressor is below the Maximum Ambient Temperature for Crankcase Heater Operation specified below.
+
+#### Field: Maximum Ambient Temperature for Crankcase Heater Operation
+
+This numeric field defines the maximum ambient temperature for crankcase heater operation in degree Celsius. The crankcase heater only operates when the air surrounding the compressor is below this maximum temperature value and the compressor is off   The ambient temperature surrounding the compressor is set by the Heat Pump:Water Heater parent object (field Compressor Location).
```

Comment:
3rd line...compressor is off   The...and end of line...Heat Pump:Water Heater (no spaces)